src/fhirtypes/common.ts

```diff
@@ -469,7 +469,9 @@ export function applyInsertRules(
         ruleSetRule.sourceInfo.appliedFile = rule.sourceInfo.file;
         ruleSetRule.sourceInfo.appliedLocation = rule.sourceInfo.location;
         if (isAllowedRule(fshDefinition, ruleSetRule)) {
-          expandedRules.push(cloneDeep(ruleSetRule));
+          const ruleSetRuleClone = cloneDeep(ruleSetRule);
+          ruleSetRuleClone.path = `${rule.path ? rule.path + '.' : ''}${ruleSetRuleClone.path}`;
```

Comment:
Not a biggie, but I feel like it's much easier to see what's going on if this is just expressed more simply as:
```ts
if (rule.path) {
  ruleSetRuleClone.path = `${rule.path}.${ruleSetRuleClone.path}`;
}
```

Reply:
I wrote this before I noticed the issue w/ `[+]` but I'm guessing that the fix for that will be somewhere in this area too?

Reply:
Yes I agree, that soft indexing isn't working as I would expect. So I will fix that, and I think it will be around this area, yes. So I will make sure however this is changed, it makes more sense than the conditional within a template string.

Reply:
This should be fixed in https://github.com/FHIR/sushi/pull/853/commits/7869bd913a5dd54af2ff7753edb617505eb96dba.